content/sensu-go/5.20/reference/tokens.md

```diff
@@ -27,15 +28,30 @@ The agent replaces any tokens with matching attributes from the entity definitio
 Invalid templates or unmatched tokens return an error, which is logged and sent to the Sensu backend message transport.
 Checks with token-matching errors are not executed.
 
-Token substitution is supported for [check definition][7] `command` attributes and [hook][8] `command` attributes.
+Token substitution is supported for [check][7], [hook][8], [handler][10], and [mutator][11] `command` attributes.
```

Comment:
Token substitution is actually not supported for handlers & mutators; we now support token substitution for assets, which handlers & mutators can rely on, via the runtime_assets.

Here's a concrete example. Let's say I have this asset definition:

```
type: Asset
api_version: core/v2
metadata:
  name: sensu-go-hello-world
  namespace: default
spec:
  builds:
  - sha512: 06665fea5b7c7ee15e7822820aa7ddb791cc6338e38444e976e601bc7d7970592e806a7b88733690a238b7325437d31f85e98ae2fe47b008ca09c86530da9600
    url: "{{ .labels.asset_url }}/sensu-go-hello-world-0.0.1.tar.gz"
```

Now, checks & hooks can use the `sensu-go-hello-world` asset as a runtime_asset, and we will do token substitution with the agent's entity. The same things applies for assets referred by mutators & handlers, except we will use the backend's entity and not the agent's entity. The use case here is that you could host your assets on different URLs depending on the datacenter etc. without having to duplicate your assets.

Reply:
Thank you -- this is really helpful. I will update the PR text and hopefully get it right.